### PR TITLE
Update colorbar tick names in sunpath gallery example

### PR DESCRIPTION
**Pull request by AdamRJensen**
- [x] Duplicate of #1998
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Comment by AdamRJensen** [comment]
@echedey-ls, @IoannisSifnaios, @RDaxini

Hi GSoC students
This PR is rather simple, but could be a good exercise for you to review

**Comment by echedey-ls** [comment]
Great @AdamRJensen ! I will review in depth later.

Handy links:
- This PR example: https://pvlib-python--2097.org.readthedocs.build/en/2097/gallery/solar-position/plot_sunpath_diagrams.html#sphx-glr-gallery-solar-position-plot-sunpath-diagrams-py
- Stable example: https://pvlib-python.readthedocs.io/en/stable/gallery/solar-position/plot_sunpath_diagrams.html#sphx-glr-gallery-solar-position-plot-sunpath-diagrams-py

**Comment by echedey-ls** [comment]
Let's add @PhilBrk8 to the whatsnew file:

```* :ghuser:`PhilBrk8` ```

File: docs/examples/solar-position/plot_sunpath_diagrams.py [hunk 1]
```diff
@@ -128,8 +142,10 @@
     label = date.strftime('%Y-%m-%d')
```
[the remark this answers is not in the document]
Comment (IoannisSifnaios) [reply docs/examples/solar-position/plot_sunpath_diagrams.py:142]: I suggest you also do that in the previous figure

File: docs/examples/solar-position/plot_sunpath_diagrams.py [hunk 2]
```diff
@@ -33,7 +33,14 @@
 # draw the analemma loops
 points = ax.scatter(np.radians(solpos.azimuth), solpos.apparent_zenith,
                     s=2, label=None, c=solpos.index.dayofyear)
```
[the remark this answers is not in the document]
Comment (IoannisSifnaios) [reply docs/examples/solar-position/plot_sunpath_diagrams.py:35]: If you apply this, do it also in the figure below

**Review by IoannisSifnaios — COMMENTED** [review]
Looks great! Just a few ideas from my side

File: docs/examples/solar-position/plot_sunpath_diagrams.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (IoannisSifnaios) [reply docs/examples/solar-position/plot_sunpath_diagrams.py:142]: I find it very annoying that I can't comment on lines that have not been changed... I guess this is a GitHub thing though! Anyway, maybe also modify the text to say that "....  The `solid` colored lines show the...."

**Review by RDaxini — COMMENTED** [review]
Thanks for the notification @AdamRJensen .
Mostly looks fine to me but two things I noticed;

1. Could we switch to 24hr format for the times in the docs? Not sure if there is a preference for pvlib docs but I think it would be a more standard way to write the times and also be consistent with the times 06--19 on the figure already.
2.  Is it possible them to align the times on the figure so that they are all above the orange 2019-06-21 line, as 19 is?

**Review by RDaxini — CHANGES_REQUESTED** [review]
Thanks for the notification @AdamRJensen .
Mostly looks fine to me but two things I noticed;

1. Could we switch to 24hr format for the times in the docs? As opposed to writing AM/PM. Not sure if there is a preference for pvlib docs but I think it would be a more standard way to write the times, easily understood, and also be consistent with the times 06--19 on the figure already.
2. Is it possible them to align the times on the figure so that they are all above the orange 2019-06-21 line, as 19 is?

**Comment by AdamRJensen** [comment]
@kandersolar I've addressed @RDaxini's review as well so this PR should be ready to merge. Might as well get it out of the way with the release of 0.11.0.

**Review by kandersolar — APPROVED** [review]

**Comment by PhilBrk8** [comment]
> Let's add @PhilBrk8 to the whatsnew file:
> 
> `` * :ghuser:`PhilBrk8` ``
> 
> EDIT: LGTM the code from a quick glance

The Whatsnew-file is what exactly?
I get notified of every change?
 - Would be down for it.
 ...even if life gets in the way a lot, I still plan on contributing more to Open Source and picked out this library to do so -

**Comment by echedey-ls** [comment]
Hey @PhilBrk8 !

> The Whatsnew-file is what exactly?

It is the changelog, a set of files that summarize the changes made to each one of the past releases and the next one. They are the source file(s) to build the following page: https://pvlib-python.readthedocs.io/en/stable/whatsnew.html

> I get notified of every change?

You won't. We mention you in the [contributors section](https://pvlib-python.readthedocs.io/en/stable/whatsnew.html#contributors). It's a way of encouraging and giving credit to contributors. You can find yourself listed on the previous link. Even though your PR wasn't merged, the original idea is still yours.

Feel free to contribute whenever you can and want, you will be welcomed here!

**Pull request merged**